### ISSUE TITLE
Join TIA Mitigation Data Sets

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#6965]
We need to join Nelson's mitigation data with AECOM's mapped data. 
_The two data sets are [here](https://drive.google.com/drive/folders/1I7lSLv3fR-nS9crk8fZBVqJrGi_rNYv3)._

**Comment by TracyLinder** [comment]
@ChrispinP can you please add an estimate? Thank you.

**Comment by ChrispinP** [comment]
All Datasets have officially been joined with clean field names and some data cleaning totally 2,337 records. Initial duplicates have been removed to drop the record count down to 1,662. I am now in the process of merging data (which must be done manually unfortunately, no Excel shortcut when it comes to combining multiple text records into multiple text columns). Once this is done I will be able to delete the remaining duplicate records and finish up the data cleaning on certain fields.

**Comment by ChrispinP** [comment]
There is a high likelihood that the historical mitigation data will need more historical cases imported. I will separate out the case data from the mitigation data and compare with what we have in Knack. Will do a case import with the missing cases so all mitigations can connect to a case.

Geo IDs do not match updated CTM/CTN data at all so figuring out if this field has any value at all. Partial matches on Mitigation Location and Coordinates would be our only way to match records with a Feature ID (Segment or Intersection ID) so that they can be connected to a geo record.

Headers and certain values have been cleaned up in preparation for import. I have also added the necessary fields to the Mitigation object in Knack. Certain values will be added/updated after import so I dont have to do extra work in Excel.

**Comment by ChrispinP** [comment]
Mitigation data as clean as its gonna get I think without fixing capitalization and grammar errors. Will begin working on the full case list to see if any cases are missing and need to still be imported into Knack.

**Comment by ChrispinP** [comment]
514 TIA Cases have been successfully imported.

This issue is now complete. Next we update the Geo records and then prep Mitigations for import thereafter.

**Comment by dianamartin** [comment]
Closing since the historical TIA case import has been completed, will create a new issue for the TIA geo location association.
